packages/compass-components/src/components/accordion.tsx

```diff
@@ -52,12 +52,14 @@ interface AccordionProps extends React.HTMLProps<HTMLButtonElement> {
   hintText?: string;
   open?: boolean;
   setOpen?: (newValue: boolean) => void;
+  onOpenCallback?: () => void;
```

Comment:
Adding callback to a callback prop name is redundant

```suggestion
  onOpen?: () => void;
```